lib/ain-contracts/src/lib.rs

```diff
@@ -8,7 +8,10 @@ pub type Result<T> = std::result::Result<T, anyhow::Error>;
 
 pub const DST20_ADDR_PREFIX_BYTE: u8 = 0xff;
 pub const INTRINSICS_ADDR_PREFIX_BYTE: u8 = 0xdf;
-
+pub const IMPLEMENTATION_SLOT: H256 = H256([
```

Comment:
Could you comment in the address here as well?

Reply:
Done: dab2d85